Src/Support/GoogleApis.Auth/OAuth2/ServiceCredential.cs

```diff
@@ -173,12 +173,24 @@ public void Initialize(ConfigurableHttpClient httpClient)
 
         #region IHttpUnsuccessfulResponseHandler
 
+        /// <summary>
+        /// Decorates unsuccessful responses, returns true if the response gets modified.

```

Comment:
I don't like adding this comment. We already have it in the interface level:
https://developers.google.com/api-client-library/dotnet/reference/1.10.0/classGoogle_1_1Apis_1_1Auth_1_1OAuth2_1_1ServiceCredential#a1083a37601642b6c3de3848c7e7a9c35
I avoided those cases in the library all time long, I don't like starting with it now.